### PR TITLE
Update wallet-js CD job

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
# Description

Simple cleanup of the wallet-js npm package publishing process

### CODE REVIEW

File: .github/workflows/js_wallet_publish.yml [hunk 1]
```diff
@@ -6,39 +6,43 @@ on:
...
-  initial_release:
-    name: Publish wasm package
-    runs-on: ubuntu-latest
-    outputs:
-      upload_url: ${{ steps.create_release.outputs.upload_url }}
-      version: ${{ steps.get_version.outputs.version }}
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
+  # initial_release:
```
Comment (minikin) [inline .github/workflows/js_wallet_publish.yml:9]: What was the motivation to comment out this job?

File: .github/workflows/js_wallet_publish.yml [hunk 2]
```diff
@@ -6,39 +6,43 @@ on:
...
+    name: Release wallet js assets
+    # needs: initial_release
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        target: [nodejs, web, bundler]
+        target: [
+                nodejs, 
+                # web,
```
Comment (minikin) [inline .github/workflows/js_wallet_publish.yml:43]: Should we uncomment `web` and `bundler`?

**Review by minikin — CHANGES_REQUESTED** [review]

File: .github/workflows/js_wallet_publish.yml (hunk 1 above)
Comment (Mr-Leshiy) [reply .github/workflows/js_wallet_publish.yml:9]: I have removed it for testing purposes, forgotten to uncomment

File: .github/workflows/js_wallet_publish.yml (hunk 2 above)
Comment (Mr-Leshiy) [reply .github/workflows/js_wallet_publish.yml:43]: At this moment it is not possible to publish a few versions of packages under the same rpm package registry (at least at that way which we are doing at this moment), so only one job can finish successfully. I would let at this moment only `nodejs` version and figure out how it is better to publish a few later.

**Review by minikin — APPROVED** [review]

**Pull request merged**